mesheryctl/internal/cli/root/system/check.go

```diff
@@ -228,11 +228,11 @@ func (hc *HealthChecker) runDockerHealthChecks() error {
 			log.Warn("!! docker-compose is not available")
 			//No auto installation of Docker-compose for windows
 			if runtime.GOOS == "windows" {
-				return errors.Wrapf(err, "please install docker-compose. Run `mesheryctl system %s` after docker-compose is installed.", hc.Options.Subcommand)
+				return errors.Wrapf(err, "please install docker-compose. Run `mesheryctl system %s` after docker-compose is installed ", hc.Options.Subcommand)
```

Comment:
I’d like to see this punctuation stay.

Reply:
Is there a mechanism that can be used to flag to ignore?

punctuation helps with legibility.

Reply:
Then revive linter must be disabled... 👀👀 Do we want to disable that linter? Or I'll add a space to avoid conflict

Reply:
Hmm. Shoot. Let's press forward as-is for the time being and see how bad the lack of punctuation is in real use.